### PR TITLE
Lando update

### PR DESCRIPTION
**Pull request by imonroe**
# READY FOR REVIEW

# Summary
- Sets up Lando stuff for localdev on Library and Press sites.
- There isn't a front-end repo yet for Press, so it doesn't include that, but it'll be easy to add.

# Review By (Date)
- 🤷 

# Urgency
- regular

# Steps to Test

1. You must have lando installed to test.  Clone this repo to a convenient place, and follow the instructions in `/docs/lando.md`
2. Verify the localdev spins up correctly.

# Affected Projects or Products
- None

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: docroot/sites/default/settings/includes.settings.php [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<?php
```
Comment (pookmish) [inline docroot/sites/default/settings/includes.settings.php]: This isn't applicable to the `default` directory.

File: composer.json [hunk 2]
```diff
@@ -176,18 +176,24 @@
...
+            "vendor/bin/blt blt:init:settings",
+            "vendor/bin/blt sws:keys"
         ],
         "pull-sul": [
             "git remote add stanford_profile git@github.com:SU-SWS/stanford_profile.git; git subtree pull --prefix=docroot/profiles/lagunita/sul_profile --squash stanford_profile 11.x"
         ],
         "pull-press": [
             "git remote add stanford_profile git@github.com:SU-SWS/stanford_profile.git; git subtree pull --prefix=docroot/profiles/lagunita/supress --squash stanford_profile 11.x"
+        ],
+        "sync-sul": [
```
Comment (pookmish) [inline composer.json]: Why not just use a blt command instead? It's already built in: `blt drupal:sync --site=library --sync-public-files`

File: lando/sul.local.settings.php [hunk 3]
```diff
@@ -0,0 +1,169 @@
+<?php
```
Comment (pookmish) [inline lando/sul.local.settings.php]: Why are any of these settings files necessary? `blt settings` already creates these using the values from `blt/local.blt.yml`. If you set up that file first, then it handles db credentials for you for every site. Add the following to `local.blt.yml`:
```
drupal:
  db:
    database: dbname
    username: drupal
    password: drupal
    host: dbhost
    port: dbport
```
Then just update the lando database to match the pattern: `[dbname]_[sitename]`

File: patches/contrib/ui_patterns_views-preview.patch [hunk 4]
```diff
@@ -59,16 +59,17 @@ index 0000000..7f1e292
 +{%- endif %}
```
Comment (pookmish) [inline patches/contrib/ui_patterns_views-preview.patch]: Not sure why there's changes to this file here. it doesn't even look like anything changed.

**Review by pookmish — CHANGES_REQUESTED** [review]

File: lando/sul.local.settings.php (hunk 3 above)
Comment (imonroe) [reply lando/sul.local.settings.php]: fixed.

File: patches/contrib/ui_patterns_views-preview.patch (hunk 4 above)
Comment (imonroe) [reply patches/contrib/ui_patterns_views-preview.patch]: fixed.

File: composer.json (hunk 2 above)
Comment (imonroe) [reply composer.json]: The syntax is slightly different (`--site=`, not `--sitename=`), and it didn't work for `supress` until there was an actual `supress` database on prod.  Should be good now.  Actually saves a different step as well.

File: docroot/sites/default/settings/includes.settings.php (hunk 1 above)
Comment (imonroe) [reply docroot/sites/default/settings/includes.settings.php]: removed.

File: composer.json (hunk 2 above)
Comment (pookmish) [reply composer.json]: I still don't see why these composer commands are desired. the same thing is done with blt.

File: lando/example.local.blt.yml [hunk 5]
```diff
@@ -8,11 +8,14 @@ project:
...
 drupal:
   db:
+    database: database
     username: drupal
     password: drupal
-    host: sulgryphon
-    database: drupal
+    host: database
     port: 3306
+multisites:
```
Comment (pookmish) [inline lando/example.local.blt.yml]: These get inherited by the `blt.yml` file so adding them here prevents new multisites from being available.

**Review by pookmish — CHANGES_REQUESTED** [review]

File: composer.json (hunk 2 above)
Comment (imonroe) [reply composer.json]: It was for convenience, since the syntax is easier, but no big deal, I removed it.

File: lando/example.local.blt.yml (hunk 5 above)
Comment (imonroe) [reply lando/example.local.blt.yml]: removed.

**Review by pookmish — APPROVED** [review]
Thanks

**Pull request merged**